blueocean-dashboard/src/main/js/components/InputStep.jsx

```diff
@@ -69,17 +70,17 @@ export default class InputStep extends Component {
      * Submit the form as "cancel" out of the state data id.
      */
     cancelForm() {
-        const { id } = this.state;
-        parameterApi.cancelInputParameter(this.state.href, id);
+        const { href, id } = this.state;
+        parameterApi.cancelInputParameter(href, id);
```

Comment:
BTW from now on I would use the logging that @tfennelly introduced before the api call for debugging reasons.

```
            if (logger.isDebugEnabled()) {
                logger.debug(`Will send cancel input call to ${href} for input step id: ${id}`);
            }
```

...and that looks so much cleaner then prefixing everything with `this.state`